.vscode/launch.json

```diff
@@ -47,6 +47,12 @@
         "${workspaceFolder}/out/**/*.js"
       ],
       "preLaunchTask": "Build"
-    }
+    },
+    {
+      "name": "PowerShell Launch Current File",
```

Comment:
I'm using this _a lot_ when testing, high time to put it in the project's config.